Content.Shared/Anomaly/Effects/Components/EntitySpawnAnomalyComponent.cs

```diff
@@ -2,52 +2,77 @@
 
 namespace Content.Shared.Anomaly.Effects.Components;
 
-[RegisterComponent]
+[RegisterComponent, Access(typeof(SharedEntityAnomalySystem))]
 public sealed partial class EntitySpawnAnomalyComponent : Component
+{
+    /// <summary>
+    /// All types of entity spawns with their settings
+    /// </summary>
+    [DataField]
+    public List<EntitySpawnSettingsEntry> Entries = new();
+}
+
+[DataDefinition, Serializable, DataRecord]
+public partial record struct EntitySpawnSettingsEntry()
```

Comment:
As below, also really needs an interface as it has like 9 proeprties in common with the tile one.